v2/internal/pkg/operator/catalog_handler.go

```diff
@@ -418,28 +442,29 @@ func isPreReleaseOfFilteredVersion(version string, chEntryName string, filteredV
 	return false
 }
 
-func handleRelatedImages(bundle declcfg.Bundle, operatorName string, copyImageSchemaMap *v2alpha1.CopyImageSchemaMap) []v2alpha1.RelatedImage {
+func handleRelatedImages(bundle declcfg.Bundle, operatorName string, copyImageSchemaMap *v2alpha1.CopyImageSchemaMap) ([]v2alpha1.RelatedImage, error) {
 	var relatedImages []v2alpha1.RelatedImage
 
 	for _, ri := range bundle.RelatedImages {
 		if strings.Contains(ri.Image, "oci://") {
-			internalLog.Warn("%s 'oci' is not supported in operator catalogs : SKIPPING", ri.Image)
-			continue
+			msg := fmt.Sprintf("image: %s 'oci' is not supported in operator catalogs SKIPPING bundle: %s for Operator: %s", ri.Image, bundle.Name, operatorName)
```

Comment:
```suggestion
			msg := fmt.Sprintf("invalid image %s: 'oci' is not supported in operator catalogs", ri.Image)
```

Reply:
The error message here should not "know" whether it's causing the bundle to be skipped or not.